.github/workflows/release.yml

```
@@ -32,8 +32,7 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - run: go install github.com/goreleaser/goreleaser/v2@latest
-      - run: goreleaser release 
+      - run: go tool goreleaser release 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)